### DISCUSSION TITLE
Single button cycling through 4 different hotkeys?

### DISCUSSION THREAD

**Discussion by IwanThomas1** [elgatosf/elgato#77] [Help]
Hello, Im looking for a BR super macro script that cycles through my 4x fancy zone window layouts.

Each layout is tied to a different hotkey:
WIN+CTRL+ALT+ 1
WIN+CTRL+ALT+ 2
WIN+CTRL+ALT+ 3
WIN+CTRL+ALT+ 4

I would like to press one button on my stream deck and have it cycle through the four hotkeys above and repeat beck to the start if it press it more than four times.

Is anyone able to help me with this?

Many thanks

**Comment by ChekTek** [comment]
Hi @IwanThomas1, 

You may have better luck finding a solution in [BarRaiders Discord](https://discord.gg/fruGhxMXcP).
